File: testprojects/csharp/benchmark/domain/Direct/Violating/DeclarationTypeCastOfArgument.cs
namespace Domain.Direct.Violating
{
	public class DeclarationTypeCastOfArgument
	{
		public void InitializeProfileInformation()
		{
            GetProfileInformation((Technology.Direct.Dao.ProfileDAO)new object());
		}

		public string GetProfileInformation(object o)
		{
			return o.ToString();
		}
	}
}

File: testresources/java/accuracy/src/domain/direct/violating/DeclarationTypeCastOfArgument.java
package domain.direct.violating;

import technology.direct.dao.ProfileDAO;

public class DeclarationTypeCastOfArgument {
	
	public String initializeProfileInformation(){
		String s;
		s = getProfileInformation((ProfileDAO) new Object());
		return s;
	}
	public String getProfileInformation(Object o){
		return o.toString();
	}
}